src/components/PhotoViewer/index.tsx

```diff
@@ -72,6 +72,34 @@ function PhotoViewer(props: Iprops) {
 
     useEffect(() => {
         if (!pswpElement) return;
+        function handleKeyUp(event) {
+            if (!isOpen) {
+                return;
+            }
+            console.log(
+                'Event: ' + event.key + ' isShiftPressedx: ' + event.shiftKey
+            );
+            switch (event.key) {
+                case 'i':
+                    if (isOpen && !showInfo) {
+                        setShowInfo(true);
+                    }
+                    break;
+                case 'Backspace':
+                    confirmTrashFile(photoSwipe?.currItem as EnteFile);
```

Comment:
the current item value doesn't get refreshed when I swipe left or right and attempt to download photo